os/pm/Kconfig

```diff
@@ -42,6 +43,60 @@ config PM_METRICS_DURATION
 
 endif
 
+comment "PM Options"
+
+config PM_NSTATE
+	int "Number of SoC sleep states"
+	default 0
+	---help---
+		Number of sleep states provided by the SoC Power module
+
+config PM_STATE_SWITCH_THRESH
+	string "PM STATE count threshold"
+	default ""
+	---help---
+		Enter values in the form: value_1,value_2,
+
+		State changes occur when the weight activity account crosses
+		threshold values for certain periods of time (time slice count).
+
+		CONFIG_PM_STATE_SWITCH_THRESH is the count for entering state xxx.
+
+		Resuming to normal state, on the other hand, is usually immediate and
+		controlled by wakeup conditions established by the platform.
+		Hence no threshold is required for normal state.
+
+		Default IDLE slices in multiples of 10 to enter subsequent idle modes from normal
+
+config PM_STATE_ENTER_THRESH
+	string "PM STATE enter threshold"
+	default ""
+	---help---
+		Enter values in the form: value_1,value_2,
+
+		State changes then occur when the weight activity account crosses
```

Comment:
what is weight activity account ?  did you intend to say count ?

Reply:
It means activity accumulator count which is based on task priorities in the previous two time slices